### PR TITLE
Add SteamLink instructions to Quest Pro page

### PR DESCRIPTION
**Pull request by kusomaigo**
+ Add SteamLink instruction to Quest Pro page and update rest of page to include references to new section.
+ Small updates to troubleshooting section
+ Small fixes across the page

Thanks lumey

### CODE REVIEW

File: docs/hardware/quest-pro.mdx [hunk 1]
```diff
@@ -15,7 +15,7 @@ Previously, a beta version of the native VRChat Quest client included built in e
 
 :::danger
 The Quest Pro has been shown to be *extremely* susceptable to sweat damage.
-There have been a non-trivial number of users who have experienced the exact same failure mode
+There have been a non-insignificant number of users who have experienced the exact same failure mode
```
Comment (regzo2) [inline docs/hardware/quest-pro.mdx:18]: double-negative (should be 'significant')

File: docs/hardware/quest-pro.mdx [hunk 2]
```diff
@@ -36,43 +36,51 @@ The basic Quest Pro setup flow:
...
 - [Oculus Link/Airlink](#oculus-linkairlink-setup)
 - [ALXR](#alxr-setup)
 - [ALVR](#alvr-setup)
 - [Virtual Desktop](#virtual-desktop-setup)
+- [SteamLink](#steamlink-setup)
 
 Oculus Link/Airlink is Meta’s own PCVR streaming solution, and is generally more straight-forward to set up (the PCVR part) and network stable, but also is limited in customization.
 ALXR/ALVR are open-source projects and thus has many for features and customizability in the PCVR connection as compared to Airlink, but also may have more bugs, quirks, and generally may be harder to use.
 Virtual Desktop is commonly seen as the "best" PCVR streaming solution, but it is not *free*, unlike all the other options.
+SteamLink is Valve's just-released PCVR steaming solution meant specifically for using Quest headsets with SteamVR, meaning setup is a breeze but the software may be prone to various bugs.
```
Comment (regzo2) [inline docs/hardware/quest-pro.mdx:50]: wouldn't say 'just released' since it is timed terminology

File: docs/hardware/quest-pro.mdx [hunk 3]
```diff
@@ -254,11 +253,55 @@ In case there is a beta you would like to try, you can switch to the beta follow
...
+:::
+
+</details>
+
 ---
 
 ## Modules
 
-There are a total of **5** different VRCFT modules that can be used for the Quest Pro.
+There are a total of **6** different VRCFT modules that can be used for the Quest Pro.
```
Comment (regzo2) [inline docs/hardware/quest-pro.mdx:304]: above sections says 5, this one says 6 (presumably because of the extra SteamLink module)

**Review by regzo2 — CHANGES_REQUESTED** [review]
Looks good, some word changes

File: docs/hardware/quest-pro.mdx (hunk 3 above)
Comment (kusomaigo) [reply docs/hardware/quest-pro.mdx:304]: 5 connection methods, 6 modules (due to ALXR having 2 different modules)

**Review by regzo2 — APPROVED** [review]
looks good for real this time

**Pull request merged**